flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/service/StandaloneFlinkService.java

```diff
@@ -134,20 +134,20 @@ protected void deleteClusterInternal(ObjectMeta meta, boolean deleteHaConfigmaps
         final String clusterId = meta.getName();
         final String namespace = meta.getNamespace();
 
-        LOG.info("Deleting Flink Standalone cluster TM resources");
+        LOG.info("Deleting Flink Standalone cluster JM resources");
         kubernetesClient
                 .apps()
                 .deployments()
                 .inNamespace(namespace)
-                .withName(StandaloneKubernetesUtils.getTaskManagerDeploymentName(clusterId))
+                .withName(StandaloneKubernetesUtils.getJobManagerDeploymentName(clusterId))
                 .delete();
 
-        LOG.info("Deleting Flink Standalone cluster JM resources");
+        LOG.info("Deleting Flink Standalone cluster TM resources");
```

Comment:
Do we need to wait until the JM is deleted before starting the TM deletion?

Reply:
I don't think that is needed